ocfweb/account/register.py

```diff
@@ -107,6 +109,23 @@ def request_account(request):
         },
     )
 
+def recommend(request):
+    real_name = request.GET.get('real_name', '')
+    first_name, last_name = real_name.split()
+    rec_lst = recommender.recommend(first_name, last_name, 3)
+    rec_dict = {}
+    for i in range(len(rec_lst)):
+        rec_dict[str(i)] = rec_lst[i] # convert i to string because JS dicts need string keys
```

Comment:
Does this even need to be a map? You can return just a list in a JSON response.

Reply:
![screenshot-docs djangoproject com-2017-11-17-09-10-42-640](https://user-images.githubusercontent.com/16551944/32959680-3b16cabc-cb77-11e7-9a15-07c8fb3cd788.png)

I wasn't all too sure whether it was okay to make this assumption.

Reply:
It might be better to return a map with a single entry, i.e. `{"recommendations": [...]}`. This is more flexible in the future event that you want to add additional fields to the output.

Reply:
Yea that sounds like a better idea